### ISSUE TITLE
setting on-click in custom modules makes the module disappear from waybar

### ISSUE THREAD

**Issue by DyslexicAtheist** [Alexays/Waybar#785]
Hi,

using this will successfully launch a full wttr.in forecast in a fullscreen terminal, however as soon as the right-click happens the custom/weather model disappears from waybar until waybar (sway) is restarted: 

```
    "custom/weather": {
        "format": "{}",
        "format-alt": "{alt}: {}",
        "format-alt-click": "click-left",
        "interval": 300,
        "return-type": "json",
        "exec": "~/.config/waybar/modules/weather.sh",
        "exec-if": "ping wttr.in -c1",
        "on-click-right": "kitty --start-as fullscreen bash -ci ~/bin/wttr|less -X"
    }
```

the weather.sh script always returns correct json: 

```
{"text":" 28 to 29°C", "alt":"Paris", "tooltip":"Paris: 28 to 29°C Partly cloudy"}
```

any ideas? 

thanks

**Comment by DyslexicAtheist** [comment]
just double-checked this to rule out it's in my scripting by using the [example](https://github.com/Alexays/Waybar/wiki/Module:-Custom#module-custom-config-return-type) 

```
"custom/weather": {
    "exec": "curl 'https://wttr.in/?format=1'",
    "interval": 3600
  }
```

adding this to what is suggested in the wiki will lead to the module disappearing in the same manner:

```
"on-click-right": "kitty --start-as fullscreen bash -ci ~/bin/wttr|less -X"
```

so either on-click actions aren't supported in custom modules or this is a bug.

**Comment by excellentname** [comment]
It's a bug, same as #739. I made a fix #778 if you want to test it.

**Comment by DyslexicAtheist** [comment]
legendary, thank you. Confirmed that this solves the problem.
